src/api/common_api.js

```diff
@@ -845,6 +845,9 @@ module.exports = {
                 create_bucket: {
                     $ref: 'common_api#/definitions/op_stats_val'
                 },
+                list_bucket: {
+                    $ref: 'common_api#/definitions/op_stats_val'
```

Comment:
as we discussed,
this schema should be refactored to something like op_name: 'string'
and value, currently it adds a lot of redundant code and a redundant maintenance

Reply:
Currently, there is no plan to change the schema, We made it this way to be explicit. 
We wanted the schema to state each op name. 
This is actually not a redundant code, but I do agree that it needs more attention when adding items to the metrics. 

We can have a discussion later on the road to determine if this should be explicit or not. 
@guymguym @dannyzaken @nimrod-becker WDYT?